docs/Calendar.md

```diff
@@ -78,4 +78,598 @@ export const App = () => (
+
+const EventList = () => (
+    <CompleteCalendar
+        ListProps={{
+            perPage: 100,
+        }}
+    >
+        <SimpleForm>
+            <TextInput source="title" />
+            <DateTimeInput source="start" />
+            <DateTimeInput source="end" />
+        </SimpleForm>
+    </CompleteCalendar>
+);
+```
+
+Check the possible values for `ListProps` in [the `<List>` component documentation](./List.html#the-list-component).
+
+### `CalendarProps`
+
+Under the hood, `<CompleteCalendar>` renders a `<Calendar>` element, which is a react-admin wrapper for Full Calendar. You can customize all the [`<Calendar>`](#calendar) and [`FullCalendar`](https://fullcalendar.io/docs#toc) props by passing a `CalendarProps` prop.
+
+For instance, to set a French locale, map non-standard event records to the expected Event type, and define boundaries to the possible events:
+
+```tsx
+import React from 'react';
+import { DateTimeInput, SimpleForm, TextInput } from 'react-admin';
+import { CompleteCalendar } from '@react-admin/ra-calendar';
+import frLocale from '@fullcalendar/core/locales/fr';
+
+const EventList = () => (
+    <CompleteCalendar
+        CalendarProps={{
+            locale: frLocale,
+            convertToEvent: (event: any): EventInput => ({
+                id: String(event.id),
+                title: event.name,
+                start: event.begin,
+                end: event.finish,
+                backgroundColor: colorForCategory(event.category),
+                borderColor: colorForCategory(event.category),
+                editable: event.can_edit,
+                url: `https://meet.jit.si/${event.jitsi_meet_id}`,
+            }),
+            validRange: {
+                start: '2017-05-01',
+                end: '2017-06-01',
+            },
+        }}
+    >
+        <SimpleForm>
+            <TextInput source="title" />
+            <DateTimeInput source="start" />
+            <DateTimeInput source="end" />
+        </SimpleForm>
+    </CompleteCalendar>
+);
+```
+
+Check the possible values for `CalendarProps` in the [`<Calendar>`](#calendar) and [`<FullCalendar>`]((https://fullcalendar.io/docs#toc) documentation.
+
+### `EditDialogProps`
+
+For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.
+
+You can customize `<EditDialog>` props like `title`, `redirect`, `onSuccess` and `onFailure` by passing a custom `EditDialogProps` prop.
+
+For instance, to customize the title of the Edit dialog:
+
+```tsx
+import React from 'react';
+import { DateTimeInput, SimpleForm, TextInput } from 'react-admin';
+import { CompleteCalendar } from '@react-admin/ra-calendar';
+
+const EventList = () => (
+    <CompleteCalendar
+        EditDialogProps={{
+            title: ({ record }) =>
+                record ? <span>Edit {record.title}</span> : null,
+        }}
+    >
```

Comment:
You seem to be missing some of the latest changes brought to the ra-calendar docs.

For instance these lines should be present: https://github.com/react-admin/ra-enterprise/blob/master/packages/ra-calendar/README.md?plain=1#L192-L195

I'll add them.